### ISSUE TITLE
Static images on home page need alt attribute

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#640]
The 4 images in the Explore section of the home page are missing alt attributes. Since they're basically decorative/described by the text below, `alt=""` is appropriate.
